Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -354,18 +331,24 @@ protected virtual void OnBeforeRender()
             }
         }
 
-        protected virtual void SetLocalTransform(Vector3 newPosition, Quaternion newRotation, bool hasTrackingStateFlagInfo, PoseDataFlags trackingStateFlags)
+
+        protected virtual void SetLocalTransform(Vector3 newPosition, Quaternion newRotation)
```

Comment:
probably want to add an [Obsolete] attribute here, pointing people to the new override.